packages/vue-components/src/panels/NestedPanel.vue

```diff
@@ -324,6 +346,17 @@ export default {
         align-items: center;
     }
 
+    .morph-display-wrapper-seamless {
+        margin-top: 5px;
+        display: flex;
+        align-items: center;
+        color: inherit;
```

Comment:
Is the difference between `.morph-display-wrapper-seamless` and `.morph-display-wrapper` only the inheritance of color? 
In that case, could we make it such that `.morph-display-wrapper-seamless` is just `color: inherit`?
So the button will always have the class `.morph-display-wrapper` and will have the class `.morph-display-wrapper-seamless`  if it is seamless

Reply:
Yep sorry about that, I have made the changes accordingly

Reply:
I've changed it now so that both the seamless and non-seamless morph wrapper would share the same basic styles, and only `morph-display-wrapper-seamless` will have the `color:inherit` property. This allows for future changes to the styles of all morph wrappers to be done in one place and allows the seamless wrapper to have the same styles as the other wrappers